easybuild/easyblocks/n/numpy.py

```diff
@@ -256,8 +259,8 @@ def sanity_check_step(self, *args, **kwargs):
         ]
         if LooseVersion(self.version) >= LooseVersion("1.10"):
             # if blas_dot symbol is not there, numpy isn't properly linked against a BLAS library
-            multiarray_so = os.path.join(self.pylibdir, 'numpy', 'core', 'multiarray.so')
-            custom_commands.append("nm %s | grep blas_dot" % multiarray_so)
+            multiarray_so = os.path.join(self.installdir, self.pylibdir, 'numpy', 'core', 'multiarray.so')
+            custom_commands.append("nm %s | grep blas_dot" % self.installdir(multiarray_so)
```

Comment:
I have PyLint set up, but wasn't paying close enough attention, my apologies. I was going to test this before merging it though. :-)